exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/Pf4jServiceLoader.java

```diff
@@ -128,18 +129,6 @@ private LoadedServiceDefinition loadDefinition(String pluginId) throws ServiceLo
     return serviceDefinition;
   }
 
-  private static ServiceArtifactId extractServiceId(String pluginId)
```

Comment:
Shan't we keep it to
1. Produce a more detailed error message,
2. Possibly, check that rid is 'Java'?